CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ All notable changes to this project will be documented in this file.
 -   #1425 : Add support for `numpy.isnan`, `numpy.isinf` and `numpy.isfinite`.
 -   #1738 : Add Python support for creating scalar sets with `{}`.
 -   #1738 : Add Python support for set method `add`.
+-   #1739 : Add Python support for set method `clear()`.
```

Comment:
Please move this to the new section of the CHANGELOG

Reply:
Okay, i will fix it in my next commit

Reply:
Fixed : 16bd679